keyboards/keyboardio/atreus/rules.mk

```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
```

Comment:
```suggestion
BOOTLOADER = atmel-dfu
```

Reply:
> All ATmega32U4 chips save for two come with atmel-dfu pre-installed. Sparkfun reflashes them with Caterina (😠) for use on Pro Micros.

And PJRC flashes them with Halfkay, and locks the bootloader. 😀